pkgs/applications/science/logic/klee/default.nix

```diff
@@ -30,19 +29,43 @@
 # Enable runtime asserts. Default false.
 , runtimeAsserts ? false
 
-# Extra klee-uclibc config.
+# Klee uclibc. Defaults to the bundled version.
+, kleeuClibc ? null
+
+# Extra klee-uclibc config for the default klee-uclibc.
 , extraKleeuClibcConfig ? {}
 }:
 
 let
+  # The chosen version of LLVM.
+  chosenLLVMPackages = let
```

Comment:
Why not use an `assert` instead of making an entirely new variable?

Reply:
Mainly because I wanted to throw a useful error.

Reply:
Fixed.

Reply:
`error: assertion '((versionAtLeast version "11") && (versionOlder version "17"))' failed` - actually a nice error :-)